src/main/java/org/swisspush/redisques/util/QueueConfiguration.java

```diff
@@ -0,0 +1,71 @@
+package org.swisspush.redisques.util;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+import java.util.List;
+
+public class QueueConfiguration {
+    
+    public static final String PROP_PATTERN = "pattern";
+    public static final String PROP_RETRY_INTERVALS = "retryIntervals";
+    
+    private String pattern;
+    
+    private List<Integer> retryIntervals;
+
+    public String getPattern() {
+        return pattern;
+    }
+
+    public List<Integer> getRetryIntervals() {
+        return retryIntervals;
+    }
+
+    public static QueueConfigurationBulider with() {
+        return new QueueConfigurationBulider();
+    }
+    
+    public JsonObject asJsonObject() {
+        JsonObject jsonObject = new JsonObject();
+        jsonObject.put(PROP_PATTERN, getPattern());
+        jsonObject.put(PROP_RETRY_INTERVALS, new JsonArray(getRetryIntervals()));
+        return jsonObject;
+    }
+    
+    public static QueueConfiguration fromJsonObject(JsonObject jsonObject) {
+        QueueConfigurationBulider builder = QueueConfiguration.with();
```

Comment:
Note that since Vert.x 3.5 (or perhaps since 3.4) you could simply do a `return sonObject.mapTo(QueueConfiguration .class);` - assuming that the JSON property names equals your Bean property names (of this QueueConfigucation class)